tests/autoyast/installation.pm

```diff
@@ -101,7 +101,9 @@ sub run {
             $num_errors++;
         }
         elsif (match_has_tag('warning-pop-up')) {
-            die 'Check the pop-up message, it is not critical so installation would continue';
+            unless (check_screen 'autoyast-known-warning') {
```

Comment:
this will wait 30 seconds in case it's not autoyast. If the autoyast-known-warning needles already contain warning-pop-up, a timeout of 0 is enough to check